### PR TITLE
corrections to zcfSeat.fail; update kinds of seats

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
Correct some details in `zcfSeat.fail()`,  and elaborate on the relations between `userSeats` and `zcfSeats`.

### CODE REVIEW

File: main/reference/zoe-api/zcfseat.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # ZCFSeat Object
 
 Zoe uses **seats** to access or manipulate offers. Seats represent active
-offers and let contracts and users interact with them. Zoe has two kinds
-of seats. **ZCFSeats** are used within contracts and with **zcf** methods.
-**UserSeats** represent offers external to Zoe and the contract.
+offers and let contracts and users interact with them. Two kinds of seats
+represent a single position, and allow interaction from within a contract
```
Comment (Tyrosine22) [inline main/reference/zoe-api/zcfseat.md:5]: The clause ", and allow interaction from within a contract and from the outside." should be deleted. It's redundant with the following sentences and is unclear.

File: main/reference/zoe-api/zcfseat.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
+and from the outside. **ZCFSeats** are used within contracts and with **zcf** methods.
+**UserSeats** represent offers external to Zoe and the contract. The two
```
Comment (Tyrosine22) [inline main/reference/zoe-api/zcfseat.md:7]:
```suggestion
**UserSeats** represent offers external to the contract. The two
```
Specifying "external to Zoe" doesn't add any useful information for the readers, correct?

File: main/reference/zoe-api/zcfseat.md [hunk 3]
```diff
@@ -36,10 +38,10 @@ const { want, give, exit } = sellerSeat.getProposal();
...
    - Returns: None.
 
-Causes the **seat** to exit, concluding its existence. All **payouts**, if any,
-are made, and the **seat** object can no longer interact with the contract.
-The **completion** argument is usually a string, but this is not required. Its
-only use is for the notification sent to the contract instance's **done()** function. 
+Causes the **seat** to exit, preventing further changes to its allocation. All **payouts**,
+if any, are made, and the **seat** object can no longer interact with the contract. The
+**completion** argument is usually a string, but this is not required. Its
+only use is for the notification sent to the corresponding userSeat's exitSubscriber.
```
Comment (Tyrosine22) [inline main/reference/zoe-api/zcfseat.md:44]:
```suggestion
only use is for the notification sent to the corresponding **UserSeat**'s **[exitSubscriber](#e-userseat-getexitsubscriber)**. 
```

File: main/reference/zoe-api/zcfseat.md (hunk 1 above)
Comment (Chris-Hibbert) [reply main/reference/zoe-api/zcfseat.md]: done

File: main/reference/zoe-api/zcfseat.md (hunk 2 above)
Comment (Chris-Hibbert) [reply main/reference/zoe-api/zcfseat.md]: yep

File: main/reference/zoe-api/zcfseat.md (hunk 3 above)
Comment (Chris-Hibbert) [reply main/reference/zoe-api/zcfseat.md]: fixed.

**Review by Tyrosine22 — APPROVED** [review]

**Pull request merged**